### ISSUE TITLE
Support archive v3

### ISSUE THREAD

**Issue by keichan34** [geolonia/pmtiles_tool#3]

**Comment by bdon** [comment]
@keichan34 please let me know if you have questions about implementing v3. 

Since it is much more efficient I would prefer if all writers can support the new format.

Thanks for developing this tool!

**Comment by keichan34** [comment]
@bdon Thanks! Yeah, we use pmtiles v2 to serve some tilesets, and the 512kb header is a bit of an issue, so I've been meaning to update to v3. I'll contact you if I have any questions about implementing the writer / reader.

**Comment by keichan34** [comment]
@bdon When I first built this tool, the PMTiles python converter was too slow and used too much memory for us, so I built a much faster and memory efficient tool. However, now that we have https://github.com/protomaps/go-pmtiles, there isn't much use in this converter anymore. I'm going to be archiving this repository in the near future -- I'll submit any improvements I come up with to the go-pmtiles repository in the future. Thanks!!
